src/App.js

```diff
@@ -50,7 +50,7 @@ class App extends Component {
 
                   <Route
                     path="/devices/"
-                    component={() => (
+                    component={props => (
```

Comment:
doesn't it complain for unused parameter. I suggest passing it down like the rest